### PR TITLE
SPOT-126 UI Fixed dendrogram overlaping lines over text

### PR DESCRIPTION
**Pull request by bryanmontesv**
Changed the way to display names on the dendrogram on DNS and Flow

### CODE REVIEW

File: spot-oa/ui/js/components/DendrogramMixin.react.js [hunk 1]
```diff
@@ -64,12 +64,23 @@ var DendrogramMixin = {
                                         .style('fill', null);
                         });
 
-      nodeEl.append('text')
-                      .attr('dx', n => n.depth===0 ? -10 : 10)
-                      .attr('dy', 3)
-                      .style('text-anchor', n => n.depth===0 ? 'end' : 'start')
-                      .attr('fill', 'black')
-                      .text(n => n.name);
+      const y1 = nodes.find(n => n.depth === 1).y;
```
Comment (bryanmontesv) [inline spot-oa/ui/js/components/DendrogramMixin.react.js]: The only way to get empty nodes is getting a date with no information.

Comment (daortizh) [inline spot-oa/ui/js/components/DendrogramMixin.react.js]: Yeah but still, we are getting empty nodes and by getting "nodes" empty we are going to get an error on the javascript console. Shouldn't we be validating that even on dates with no information?

Comment (bryanmontesv) [inline spot-oa/ui/js/components/DendrogramMixin.react.js]: We actually are validating dates, the error couldn't happen anyways unless you manually set nodes empty

**Comment by LedaLima** [comment]
+1

**Comment by raypanduro** [comment]
+1

**Pull request closed without merging**